src/pages/GetStarted/GetStartedTab/index.tsx

```diff
@@ -83,7 +83,7 @@ export class SamplesListTab extends React.PureComponent<Props, State> {
           style={{ background: '#f0f0f0' }}
         >
           <PageSection variant={PageSectionVariants.light}>
-            <Form>
+            <Form onSubmit={e => e.preventDefault()}>
```

Comment:
As far as I understand with this click on Enter will do nothing.
Ideally, it should emulate the click on Create&Open button.

Reply:
I have fixed it.